lib/resources/dstu2/document_reference_create.yaml

```diff
@@ -0,0 +1,130 @@
+---
```

Comment:
+1 to matching the official spec.  If we use different values we should add a note explaining the difference